lib/tagger.rb

```diff
@@ -0,0 +1,22 @@
+class Tagger
+  def self.add_tags(content_id, taxon_ids, do_tag = true)
+    link_content = Services.publishing_api.get_links(content_id)
+    version = link_content['version']
+    existing_taxon_ids = link_content.dig('links', 'taxons') || []
+    return [] if  (taxon_ids - existing_taxon_ids).empty?
```

Comment:
I would extract this line out to a function that explains what it's doing.

Reply:
done